src/net/throttle_node.h

```diff
@@ -28,8 +28,7 @@ class ThrottleNode {
   void                clear_quota()                   { m_quota = 0; }
   void                set_quota(uint32_t q)           { m_quota = q; }
 
-  iterator            list_iterator()                 { return m_listIterator; }
-  const_iterator      list_iterator() const           { return m_listIterator; }
+  iterator            list_iterator() const           { return m_listIterator; }
```

Comment:
This changes the permissions, e.g. it allows us to get mutable objects from const parent which is contrary to intent.